devicon.json

```diff
@@ -1004,6 +1004,26 @@
             }
         ]
     },
+    {
+        "name": "firebase",
+        "tags": [
+            "auth",
+            "hosting",
+            "storage",
+            "cloud"
+        ],
+        "versions": {
+            "svg": [
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain"
```

Comment:
Can you add the "plain-wordmark" to the `font` list please? Since it's a "plain" icon, it should be able to be converted to icons.